### DISCUSSION TITLE
Seamless tooltip arrow border

### DISCUSSION THREAD

**Discussion by nandcello** [chakra-ui/chakra-ui#4724] [Q&A]
How do I match the border of the tooltip to also have it on the arrow? (without the tooltip content border going through the arrow). Not sure how to target the tooltip arrow's border.

Something like this 
![image](https://user-images.githubusercontent.com/2021570/133627015-5f0b89e1-bc7a-425a-819b-985ba0fc00cf.png)

**Comment by primos63** [comment]
Not sure exactly what you are looking for. Perhaps something like this?

```jsx
<Tooltip hasArrow label="Search places" bg="white" color="black" border='1px solid' borderColor='red.500' sx={{
  '.chakra-tooltip__arrow-wrapper .chakra-tooltip__arrow': {
      top: '1px !important',
      borderRight: '1px solid',
      borderRightColor: 'red.500',
      borderBottom: '1px solid',
      borderBottomColor: 'red.500',
    }
}}>
  <SearchIcon />
</Tooltip>
```
Which will produce
![Example](https://user-images.githubusercontent.com/2768849/133910082-02eb8a3c-256b-4108-9154-616f87c370db.png)
